src/ImageSequencer.js

```diff
@@ -197,25 +197,54 @@ ImageSequencer = function ImageSequencer(options) {
     else modulesdata = modules[name][1];
     return modulesdata;
   }
-  
+
+  // Strigifies the current sequence
   function toString(step) {
     if(step) {
       return stepToString(step);
     } else {
       return copy(this.images.image1.steps).map(stepToString).slice(1).join(',');
     }
   }
-  
+
+  // Stringifies one step of the sequence
   function stepToString(step) {
     let inputs = copy(modulesInfo(step.options.name).inputs);
     inputs = inputs || {};
-    
+
     for(let input in inputs) inputs[input] = step.options[input] || inputs[input].default;
-    
+
     var configurations = Object.keys(inputs).map(key=>key + ':' + inputs[key]).join(',');
     return `${step.options.name}(${configurations})`;
   }
-  
+
+  // blend(blend:function(r1,%20g1,%20b1,%20a1,%20r2,%20g2,%20b2,%20a2)%20{%20return%20[%20r1,%20g2,%20b2,%20a2%20]%20})
+  // crop(X:10,y:20,z:40),invert()
+  // Coverts stringified sequence into JSON
+  function toJson(str){
+    let steps = str.split('),');
+    steps.push(steps.splice(-1)[0].slice(0,-1));
+    return steps.map(stepToJson);
+  }
+  // Converts one stringified step into JSON
+  function stepToJson(str){
```

Comment:
Maybe `stepStringToJson()`? for extra clarity

Reply:
And can we write a test for this function? Thank you!!!

Reply:
@jywarren yeah I'l add the tests now!!